--- a/chrome/content/scripts/readability.js
+++ b/chrome/content/scripts/readability.js
@@ -334,9 +334,6 @@
         articleFooter.id = "readFooter";
         articleFooter.innerHTML = [
         "<div id='rdb-footer-print'>Excerpted from <cite>" + document.title + "</cite><br />" + window.location.href + "</div>",
-        "<div id='rdb-footer-wrapper'>",
-'<a href="http://readability.com/learn-more" target="_top"><div id="legacy-bkmk"><img width="99" height="147" alt="Devices" src="chrome://readability/content/images/footer_devices.1298660361.png"><div class="block-wrap"><div class="inner"><h4>Sign Up &amp; Read Comfortably&mdash;Anytime, Anywhere</h4><p>A subscription to Readability offers great features for mobile reading, saving articles for later and supporting the writers you enjoy. <span class="main-link">Learn More »</span></p></div></div></div></a>',
-             "</div>",
         "</div>"].join('');
                 
         return articleFooter;
